packs/spell-effects/spell-effect-angel-form.json

```diff
@@ -0,0 +1,644 @@
+                                        ]
+                                    }
+                                },
+                                "tempHP": 40
+                            }
+                        },
+                        {
+                            "start": 9,
+                            "value": {
+                                "size": "lg",
+                                "skills": {
+                                    "athletics": {
+                                        "modifier": 33
+                                    }
+                                },
+                                "strikes": {
+                                    "scimitar": {
+                                        "damage": {
+                                            "modifier": 20
+                                        },
+                                        "modifier": 30
+                                    }
+                                },
+                                "tempHP": 60
+                            }
+                        }
+                    ],
+                    "field": "item|system.level.value"
+                }
+            },
+            {
+                "damageType": "fire",
+                "diceNumber": 2,
+                "dieSize": "d6",
+                "key": "DamageDice",
+                "predicate": [
+                    "angel-form:balisse"
+                ],
+                "selector": "scimitar-damage"
+            },
+            {
+                "hasHands": true,
+                "key": "BattleForm",
+                "overrides": {
+                    "senses": {
+                        "darkvision": {}
+                    },
+                    "speeds": {
+                        "fly": 40,
+                        "land": 30
+                    },
+                    "strikes": {
+                        "fist": {
+                            "ability": "str",
+                            "baseType": "fist",
+                            "category": "unarmed",
+                            "damage": {
+                                "damageType": "bludgeoning",
+                                "dice": 2,
+                                "die": "d6"
+                            },
+                            "img": "icons/skills/melee/unarmed-punch-fist.webp",
+                            "label": "PF2E.Weapon.Base.fist",
+                            "traits": [
+                                "agile",
+                                "holy",
+                                "magical",
+                                "unarmed"
+                            ]
+                        },
+                        "piercing-hymn": {
+                            "category": "unarmed",
+                            "damage": {
+                                "damageType": "sonic",
+                                "dice": 1,
```

Comment:
Some of these damages may need to be looked it. For example, I believe this is now 3d6?
<https://paizo.com/pathfinder/faq> Secrets of Magic Errata (Fall 2024, 2nd Printing)